### ISSUE TITLE
Update template packages

### ISSUE THREAD

**Issue by cmeeren** [Zaid-Ajaj/Feliz#107]
Several npm packages in the template are outdated.

**Comment by Zaid-Ajaj** [comment]
Really, I though I had some quite recent stuff. Which packages need to be updated?

**Comment by cmeeren** [comment]
```
> dotnet new feliz
> npm install
> npm outdated

Package              Current  Wanted  Latest  Location
@babel/core            7.6.4   7.7.0   7.7.0  global
@babel/polyfill        7.6.0   7.7.0   7.7.0  global
@babel/preset-env      7.6.3   7.7.1   7.7.1  global
@babel/runtime         7.6.3   7.7.1   7.7.1  global
copy-webpack-plugin    4.6.0   4.6.0   5.0.4  global
core-js                3.3.4   3.3.6   3.3.6  global
fable-compiler         2.4.7   2.4.9   2.4.9  global
sass                  1.23.1  1.23.3  1.23.3  global
webpack-cli            3.3.9  3.3.10  3.3.10  global
```

![image](https://user-images.githubusercontent.com/7766733/68275408-d3045a80-006b-11ea-87ee-d3c5f4bcb489.png)

**Comment by cmeeren** [comment]
Strange. When creating the Feliz.MateiralUI demo app, I also saw that react and react-dom was at 16.9 and that 16.11 was available, though it seems the template's lock file has been updated in the meantime to install 16.11.

**Comment by Zaid-Ajaj** [comment]
Updated and published template as of v0.2.15 :rocket:
